packages/plugins/users-permissions/admin/src/components/BoundRoute/index.js

```diff
@@ -43,13 +43,13 @@ function BoundRoute({ route }) {
             borderBottomRightRadius: 0,
           }}
         >
-          <Text bold>{method}</Text>
+          <Typography bold>{method}</Typography>
```

Comment:
The bold props doesn't exist on the typography component. Should be fontWeight="semiBold" IMO